image/spec/README.md

```diff
@@ -0,0 +1,47 @@
+# Docker Image Specification v1.
+
+This directory contains documents about Docker Image Specification v1.X.
+
+Distribution and archival specifications such as file layouts and manifests are no longer used
+in Moby and Docker, except in `docker save` and `docker load`.
+
+However, v1 Image JSON (`application/vnd.docker.container.image.v1+json`) has been still widely
+used and officially adopted in [V2 manifest](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md)
+and in [OCI Image Format Specification](https://github.com/opencontainers/image-spec), which is
+the current industry's standard image specification. 
```

Comment:
No need for value statements. Just provide facts.

Reply:
done